android-ktx/src/main/java/com/mirego/trikot/viewModels/SliderViewModelBinder.kt

```diff
@@ -0,0 +1,48 @@
+package com.mirego.trikot.viewmodels
+
+import android.os.Build
+import android.widget.SeekBar
+import androidx.core.view.ViewCompat
+import androidx.databinding.BindingAdapter
+import com.mirego.trikot.streams.android.ktx.observe
+import com.mirego.trikot.streams.reactive.just
+import com.mirego.trikot.viewmodels.LifecycleOwnerWrapper
+import com.mirego.trikot.viewmodels.SliderViewModel
+import com.mirego.trikot.viewmodels.mutable.MutableSliderViewModel
+
+object SliderViewModelBinder {
+    val noSliderViewModel =
+        MutableSliderViewModel().apply { hidden = true.just() } as SliderViewModel
+
+    @JvmStatic
+    @BindingAdapter("view_model", "lifecycleOwnerWrapper")
+    fun bind(
+        seekBar: SeekBar,
+        sliderViewModel: SliderViewModel?,
```

Comment:
@martingagnon I always wondered why we can bind nullable viewModels, which requires the `noSliderViewModel` to exsit? Why is it so?

Reply:
Honnestly, i'm not sure either. I've had a case in a previous project where the viewmodel was nullable therefore, my binder was built adequately, but I'm not sure why we'd need this here.

Reply:
View models can be null if the variable was not set before the binding is executed.

Reply:
I most cases it is never null but in some cases I've seen this happen with viewModel restoration on a config change, such as supporting screen resize. viewModel would be null and then quickly set back to a value after that and there was nothing we could do to prevent that, as execution ordering was orchestrated by the android framework.

Reply:
@npresseault I see 👍